scripts/scil_correct_gradients.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Validate and correct b-vectors using a fiber coherence index as presented
+in Schilling et al, 2019. The script takes as input the principal direction
+at each voxel, the b-vectors and the fractional anisotropy map and outputs
+a corrected b-vectors file.
+
+A typical pipeline could be:
+>>> scil_compute_dti_metrics.py dwi.nii.gz bval bvec --not_all --fa fa.nii.gz
+    --evecs peaks.nii.gz
+>>> scil_correct_gradients.py bvec peaks_v1.nii.gz fa.nii.gz bvec_corr
```

Comment:
Would it work if peaks come from a fODF? I can test. Should we mention something about this?

Reply:
I have my answer, I just tested and it breaks...
 File "/Users/desm2239/Research/Source/scilpy/scilpy/reconst/fiber_coherence.py", line 43, in compute_fiber_coherence_table
    index = compute_fiber_coherence(directions.dot(t), values)
ValueError: shapes (136,184,132,15) and (3,3) not aligned: 15 (dim 3) != 3 (dim 0)

Reply:
mrconvert peaks.nii.gz peaks_v1.nii.gz -coord 3 0:2 can do the trick... and just get the first peak of the fODF field.

Reply:
Yeah there needs to be only one peak par voxel, I will raise a parser error if the shape of the last axis of the 4D volume is different than 3.

Reply:
Or should we run if first 3 floats? And just raise a warning?
That way it would work with fODFs peaks.

Reply:
I'll add a script to extract the peak associated to the maximum value for fODF, this way we can use the output as input for this script

Reply:
I did not add an additional script after all. I just added a step to extract the principal direction from a volume with more than one direction for each voxel, see fa38a53.